### ISSUE TITLE
GANs L1.16 A Trained GAN

### ISSUE THREAD

**Issue by sahiljuneja** [udacity/dlnd-issue-reports#273]
Some general suggestions on this lesson that I hope can be improved -

- Discussion on the loss plot. Why is the loss so high? Is it the hyperparameters or just the number of epochs required to lower the loss? Why it's so spiky? What it's supposed to look like? What this current plot signifies in terms of the model and training? Being able to answer these questions would help a lot in understanding the concepts better and relating to the theory.

- When showing the generated numbers after every 10 epochs, the results don't progressively get better based on the video. A discussion on why that happens, and how we eventually get a proper model with properly generated images would help. Dumb question, I know, but looking at that I can't intuitively picture (no pun intended) what the training progress is supposed to look like.

- The code for generating new latent vectors using a random uniform distribution is the same as used for the actual training (I know each time there would be a random distribution). Perhaps a different kind of latent vector for comparison purposes? Or you think that's better if we test those out ourselves? I guess that would be better, but open to your thoughts on that.

**Comment by mcleonard** [comment]
Hey there! I'm going to collect all your feedback in a google doc since it is more general and not a quick fix.
